frontend/src/views/dashboard/MyProblems.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+import { MainHeaderText, SecondaryHeaderText } from '../../components/core/Text';
+import ProblemCard from '../../components/card/ProblemCard';
+import { TextLink } from '../../components/core/Link';
+import { useAppSelector } from '../../util/Hook';
+import {
+  CenteredContainer,
+  FlexHorizontalContainer,
+  FlexLeft,
+  RelativeContainer
```

Comment:
```suggestion
  RelativeContainer,
```
nit: Add ending comment to remove the "missing comma dangle" warning. (Sorry, I only saw this after I finished my review, which is why I'm adding this comment after.)

Reply:
Sg - I think this is outdated now since I included this change in an earlier commit, so I'll resolve now